xml_converter/generators/code_generator.py

```diff
@@ -267,6 +267,35 @@ def load_input_doc(self, dir_path: str) -> None:
                 content=document.content
             )
     
+    def write_parseable(self, output_directory: str) -> None:
+        file_loader = FileSystemLoader('cpp_templates')
+        env = Environment(loader=file_loader)
+        template = env.get_template("parseabletemplate.hpp")
+        attributenames: Dict[str,str] = {}
+        
+        for filepath in self.data.keys():
+            filename = os.path.basename(filepath)
+            attributenames[filepath]= filename.split(".md")[0]
+     
+        pages = ["Category","Icon","Trail"]
+        
+        for page in pages:
+            content: Dict[str,str] = {}
+            metadata: Dict[str,List[str]] = {}
+            
+            for attributename in attributenames:
+                metadata[attributename] = self.data[attributename].metadata
```

Comment:
`metadata` is a `Dict[str, List[str]]` and their values is trying to be set to `self.data[attributename].metadata` which is a `Dict[str, Any]`. This seems to also be an error in `write_webdocs()` on line 330 so it was probably copied from there back when metadata was still an `Any` and would not trigger type check issues.

Reply:
metadata has now been updated to be `Dict[str, List[str]]` in all instances